docs/https_source_interface.rst

```diff
@@ -104,9 +104,10 @@ following: ::
     securedrop_app_https_certificate_chain_src: ca.crt
 
 The filenames should match the names of the files provided to you by DigiCert,
-and should be saved inside the ``install_files/ansible-base/`` directory. Then rerun
-the configuration: ::
+and should be saved inside the ``install_files/ansible-base/`` directory. You'll
+reactivate the virtual environment before running the Ansible playbook: ::
```

Comment:
The audience we should be catering for in our documentation is a SecureDrop administrator that is not a developer. They shouldn't be expected to know what a virtual environment or an Ansible playbook is, as these things should be abstracted away by `securedrop-admin`. So, let's remove the diff in these two lines. Btw note that `./securedrop-admin install` does reactivate the virtual environment without needing to run `./securedrop-admin setup` first.

However, I think it's a fine practice for admins to run `./securedrop-admin setup` after updating SecureDrop locally in case there are any changes to packages (e.g. #2592 installs `dnsutils` locally as a workaround for #2591).